docs/lts.md

```diff
@@ -1,14 +1,61 @@
 ## Long Term Support
-<a id="lts"></a>
 
-Pino's Long Term Support (LTS) is tied to the [Node.js LTS policy].
-Major versions of Pino, "X" releases in X.Y.Z of [semantic versioning] nomenclature, will
-be issued as close as possible to the release of a new major LTS release of
-Node.js (subject to the Pino maintainers's capacity). Pino makes no guarantees
-as to the operability of the library on unsupported releases of Node.js.
+Pino's Long Term Support (LTS) is provided according to the schedule laid
+out in this document:
 
-Pino will not remove support for a deprecated Node.js release via a minor,
-or patch, release of Pino.
+1. Major releases, "X" release of [semantic versioning][semver] X.Y.Z release
+   versions, are supported for a minimum period of six months from their release
+   date. The release date of any specific version can be found at
+   [https://github.com/pino/pino/releases](https://github.com/pino/pino/releases).
 
-[semantic versioning]: https://semver.org/
-[Node.js LTS policy]: https://github.com/nodejs/Release
+1. Major releases will receive security updates for an additional six months
+   from the release of the next major release. After this period
+   we will still review and release security fixes as long as they are
+   provided by the community and they do not violate other constraints,
+   e.g. minimum supported Node.js version.
+
+1. Major releases will be tested and verified against all Node.js
+   release lines that are supported by the
+   [Node.js LTS policy](https://github.com/nodejs/Release) within the
+   LTS period of that given Fastify release line. This implies that only
+   the latest Node.js release of a given line is supported.
+
+A "month" is defined as 30 consecutive days.
+
+> ## Security Releases and Semver
+>
+> As a consequence of providing long-term support for major releases, there
+> are occasions where we need to release breaking changes as a _minor_
+> version release. Such changes will _always_ be noted in the
+> [release notes](https://github.com/pino/pino/releases).
+>
+> To avoid automatically receiving breaking security updates it is possible to use
+> the tilde (`~`) range qualifier. For example, to get patches for the 3.15
+> release, and avoid automatically updating to the 3.16 release, specify
+> the dependency as `"pino": "~6.1.x"`. This will leave your application vulnerable,
```

Comment:
```suggestion
> the tilde (`~`) range qualifier. For example, to get patches for the 6.1
> release, and avoid automatically updating to the 6.1 release, specify
> the dependency as `"pino": "~6.1.x"`. This will leave your application vulnerable,
```